src/cljs/rems/fields.cljs

```diff
@@ -15,6 +15,20 @@
 (defn field-name [field]
   (str "form-" (getx field :form/id) "-field-" (getx field :field/id)))
 
+(defn info-collapse [info-id aria-label-text focus-when-collapse-opened body-text]
+  [:<> [:button.info-button.btn.btn-link
+        {:data-toggle "collapse"
+         :href (str "#" (str info-id "-collapse"))
+         :aria-label aria-label-text
+         :aria-expanded "false"
+         :aria-controls (str info-id "-collapse")}
+        [:i.fa.fa-info-circle]]
+   [:div.field-info.collapse {:id (str info-id "-collapse")
```

Comment:
The class should ideally match the component name i.e. `.info-collapse`. NB: you have to change the CSS selectors to match!

Reply:
this seems done now

Reply:
resolved